cli/internal/cmd/upgradeapply_test.go

```diff
@@ -211,7 +211,7 @@ func TestUpgradeApply(t *testing.T) {
 			},
 			fh: fsWithStateFileAndTfState,
 		},
-		"no tf state, skip infrastructure upgrade": {
+		"no tf state, but existing admin conf file": {
```

Comment:
why does this cause an error? the terraform module doesn't store it's TF state in this folder but it will have an admin file

Reply:
Previously, we had the Code set up to simply assume self-managed infrastructure if the workspace was not clean, but no tf state existed.
Since we now want to create new resources if the state does not exist, we can't simply continue if the workspace is not clean.
Maybe this test case would be better if placed in `TestValidateInputs`

Reply:
This test case already exists in `TestValidateInputs`, removing it from here.